### PR TITLE
docs(docs_data_analysis): update and improvements

### PR DESCRIPTION
**Pull request by Yalyalieva**
version v0.7.0 and improvements

### CODE REVIEW

**Comment by Yalyalieva** [comment]
I added snippets in the form as they are in citros web.

File: docs_data_analysis/data_access/index.md
[the remark this answers is not in the document]
Comment (Yalyalieva) [reply docs_data_analysis/data_access/index.md:1]: fixed

File: docs_data_analysis_versioned_docs/version-v0.7.0/documentation/error_analysis/citros_data_array.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+)
+```
+
+
+<details>
+  <summary>Description</summary>
+
+Show the predictions based on the results of the regression solution, neural net or gaussian mixture model.
+
+---
```
[the remark this answers is not in the document]
Comment (Yalyalieva) [reply docs_data_analysis_versioned_docs/version-v0.7.0/documentation/error_analysis/citros_data_array.md]: now parameters are in the tables

**Comment by mariaBer** [comment]
@Yalyalieva Please capitalize fields names in tables

**As for new additions** - they should be added in a different PR, since these should be examined as well, however they should not hold the updates we are working on this past week.
Either add them in a new branch in a totally different PR or revert, stage the changes until this last PR will be finally approved

**Comment by Yalyalieva** [comment]
@mariaBer new additions - you mean snippets? They were in the structure schema that you provided me. You want me to delete them?

**Comment by mariaBer** [comment]
I added the final structure we should have. since we had a lot of conversations about it and wanter to summarize it.

Yes, I mean remove both Snippets and installation for THIS PR.
You can add the snippets as a place holder without the content inside,  but I don't want to hold off any updates in this structure.

The snippets and installation addition (and any new additions in the future should be added as a different PR,  in an updated branch)

**Comment by Yalyalieva** [comment]
Which installations?

**Comment by mariaBer** [comment]
the one added in the main page

Basically, this PR is to examine the same thing we examined passed week.
any new additions should be added in a different PR (and in an updated branch)

File: docs_data_analysis/documentation/data_access/citros_dict.md
Comment (mariaBer) [inline docs_data_analysis/documentation/data_access/citros_dict.md:1]: Capitalize fields names in tables - relevant to all tables throughout the docs

File: docs_data_analysis/data_access/index.md [hunk 2]
```diff
@@ -3,11 +3,10 @@ sidebar_position: 1
 sidebar_label: 'Overview'
 hide_title: true
 ---
+## Data access
```
Comment (mariaBer) [inline docs_data_analysis/data_access/index.md:6]: First title should be #

File: docs_data_analysis/documentation/data_access/references.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+
+
+
+
+
+    
+## Class `Ref` {#citros_data_analysis.data_access.references.Ref}
```
Comment (mariaBer) [inline docs_data_analysis/documentation/data_access/references.md:17]: First title should be #First title should be #

File: docs_data_analysis/error_analysis/index.md [hunk 4]
```diff
@@ -3,10 +3,9 @@ sidebar_position: 1
 sidebar_label: 'Overview'
 hide_title: true
 ---
+## Error analysis
```
Comment (mariaBer) [inline docs_data_analysis/error_analysis/index.md:6]: First title should be #

File: docs_data_analysis/index.md [hunk 5]
```diff
@@ -1,11 +1,60 @@
 ---
 sidebar_position: 1
-sidebar_label: 'Package overview'
+sidebar_label: 'Package Overview'
 hide_title: true
 ---
+## Data Analysis Overview
```
Comment (mariaBer) [inline docs_data_analysis/index.md:6]: First title should be #

File: docs_data_analysis/validation/index.md [hunk 6]
```diff
@@ -3,10 +3,9 @@ sidebar_position: 1
 sidebar_label: 'Overview'
 hide_title: true
 ---
+## Validation
```
Comment (mariaBer) [inline docs_data_analysis/validation/index.md:6]: First title should be #

**Review by mariaBer — CHANGES_REQUESTED** [review]
@Yalyalieva 

- [ ] First title should be # 
- [ ] All Table fields should be capitalized

File: docs_data_analysis/documentation/data_access/references.md (hunk 3 above)
Comment (Yalyalieva) [reply docs_data_analysis/documentation/data_access/references.md:17]: for documentation files that works in a bad way - in docusaurus the title with one '#' could not be used as references so the referenerce to the constructor of the class is missed. I make titles with one '#'. I can rename the constructors if you don't want the repetitions of the lines - for example from "Class `Ref`" to "Constructor `Ref`" (but it looks a little unusual for the documentation, the constructor usually goes with 'class' title) or change titles, for example "Documentation for the class Ref"

File: docs_data_analysis/error_analysis/index.md (hunk 4 above)
Comment (Yalyalieva) [reply docs_data_analysis/error_analysis/index.md]: +

File: docs_data_analysis/index.md (hunk 5 above)
Comment (Yalyalieva) [reply docs_data_analysis/index.md]: +

File: docs_data_analysis/validation/index.md (hunk 6 above)
Comment (Yalyalieva) [reply docs_data_analysis/validation/index.md]: +

File: docs_data_analysis/documentation/data_access/citros_dict.md
Comment (Yalyalieva) [reply docs_data_analysis/documentation/data_access/citros_dict.md:1]: +

File: docs_data_analysis/data_access/index.md (hunk 2 above)
Comment (Yalyalieva) [reply docs_data_analysis/data_access/index.md]: done

File: docs_data_analysis/documentation/data_access/references.md (hunk 3 above)
Comment (mariaBer) [reply docs_data_analysis/documentation/data_access/references.md:18]: What you did is OK, we have a title and we have the class itself.

File: docs_data_analysis/data_access/getting_started.md [hunk 7]
```diff
@@ -1,12 +1,11 @@
...
 toc_max_heading_level: 4
-sidebar_label: 'Getting started'
+sidebar_label: 'Getting Started'
 hide_title: true
-# description: Examples of the data_access module
+description: 'Getting started'
 sidebar_position: 1
 ---
-
-## Data Access
```
Comment (mariaBer) [inline docs_data_analysis/data_access/getting_started.md:9]: Title should be as in sidebar

File: docs_data_analysis/error_analysis/getting_started_query_data.md [hunk 8]
```diff
@@ -1,12 +1,11 @@
...
-sidebar_label: 'Getting started: query data'
+sidebar_label: 'Getting Started: Query Data'
 hide_title: true
-# description: Examples of the error_analysis module
 sidebar_position: 1
+description: 'Quick start'
 ---
-
-# Error analysis
+# Error Analysis
```
Comment (mariaBer) [inline docs_data_analysis/error_analysis/getting_started_query_data.md:8]: Title should be as in sidebar

File: docs_data_analysis/error_analysis/index.md [hunk 9]
```diff
@@ -3,10 +3,9 @@ sidebar_position: 1
 sidebar_label: 'Overview'
 hide_title: true
 ---
+# Error analysis
```
Comment (mariaBer) [inline docs_data_analysis/error_analysis/index.md:6]: Title should be as in sidebar

File: docs_data_analysis/validation/getting_started.md [hunk 10]
```diff
@@ -1,11 +1,10 @@
...
 toc_max_heading_level: 4
-sidebar_label: 'Getting started'
+sidebar_label: 'Getting Started'
 hide_title: true
-# description: Examples of the validation module
 sidebar_position: 1
+description: 'Quick start'
 ---
-
 # Validation
```
Comment (mariaBer) [inline docs_data_analysis/validation/getting_started.md:8]: Title should be as in sidebar

File: docs_data_analysis/data_access/index.md [hunk 11]
```diff
@@ -3,11 +3,10 @@ sidebar_position: 1
 sidebar_label: 'Overview'
 hide_title: true
 ---
+# Data access
```
Comment (mariaBer) [inline docs_data_analysis/data_access/index.md:6]: Title should be as in sidebar

**Review by mariaBer — CHANGES_REQUESTED** [review]

File: sidebarsDataAnalysis.js [hunk 12]
```diff
@@ -27,11 +27,11 @@ module.exports = {
         {
           type: 'category',
           label: 'Data access',
```
Comment (mariaBer) [inline sidebarsDataAnalysis.js:29]: Capitalize

File: sidebarsDataAnalysis.js [hunk 13]
```diff
@@ -42,18 +42,19 @@ module.exports = {
...
               label: 'Documentation',
-              items: ['documentation/data_access/citros_db', 'documentation/data_access/citros_dict']
+              items: ['documentation/data_access/citros_db', 'documentation/data_access/citros_dict', 
+                      'documentation/data_access/references']
             },
           ]
         },
         {
           type: 'category',
           label: 'Error analysis',
```
Comment (mariaBer) [inline sidebarsDataAnalysis.js:52]: Capitalize

**Review by mariaBer — CHANGES_REQUESTED** [review]

**Pull request merged**